src/test/kotlin/note/mothers/TitleMother.kt

```diff
@@ -7,6 +7,7 @@ class TitleMother {
     companion object {
         private const val PRIMITIVE_TITLE = "Buy Milk"
         fun getValidTitle() = Title(PRIMITIVE_TITLE)
+        fun getValidTitleFrom(title: String) = Title(title)
```

Comment:
You can't ensure that the string that the user passes to this mother is valid. If they want a valid title they have to call getValidTitle(). Instead, if they want a valid title that isn't the same as getValidTitle(), it should be a method like for example the one above:
```kotlin
fun getDifferentValidTitleFrom(Title t) = Title(t.title + "more title")
```
Here, we ensure that the title will continue to be valid (since, the title that the user passes as a parameter is already generated by the mother.